atom/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -378,6 +389,14 @@ AVMediaType ParseMediaType(const std::string& media_type) {
   }
 }
 
+std::string SystemPreferences::GetAccentColor() {
+  NSColor* sysColor = nil;
+  if (@available(macOS 10.14, *))
+    sysColor = [NSColor controlAccentColor];
```

Comment:
you need to forward declare this as it's macOS SDK 10.14+ only
```
@interface NSColor (MojaveSDK)
@property (class, strong, readonly) NSColor *controlAccentColor NS_AVAILABLE_MAC(10_14);
@end
```

Reply:
done!